README.md

acedmicabhishek opened the pull request:
> added readme on root directory with some starter information

acedmicabhishek committed:
Create README.md

marios-pz commented:
<img width="857" height="893" alt="image" src="https://github.com/user-attachments/assets/f7fbbfa9-5ddc-4f6d-87e3-66f7234b09d5" />

vaporwavie commented:
The point of a README file is to create a channel between the repo owner and possible contributors. No point in having it AI-written. Especially if most of the content is word salad.

sammwyy commented:
Ai slop + there is already a README. Did you actually see the repository before writing this?

acedmicabhishek commented:
> Ai slop + there is already a README. Did you actually see the repository before writing this?

I saw the repository, thought it would be better if we have readme at root directory
And Using AI for readme ain't a problem as long as we handled it properly.

acedmicabhishek commented:
> <img width="857" height="893" alt="image" src="https://github.com/user-attachments/assets/f7fbbfa9-5ddc-4f6d-87e3-66f7234b09d5" />
> 

I mean seriously I'm not an English major lol.
I use AI for writing basic stuff all the time, it writes better and modular than me

acedmicabhishek committed:
Update README.md

> people said using ai for writing readme is cringe 
> so i updated a dumber version of earlier commit
> because i suck at writing
> obviously pewd aint going to merge this
> this is for stupid meme

acedmicabhishek committed:
Update README.md

acedmicabhishek committed:
Update README.md

acedmicabhishek commented:
> people said using ai for writing readme is cringe 
> so i updated a dumber version of earlier commit
> because i suck at writing
> obviously pewd aint going to merge this
> this is for stupid meme
<img width="272" height="240" alt="Screenshot From 2025-09-01 21-36-41" src="https://github.com/user-attachments/assets/b719e4ca-75c7-43f5-a76e-d1f26c76e9f1" />

supervoidcoder commented:
> > <img alt="image" width="857" height="893" src="https://private-user-images.githubusercontent.com/30930688/484221274-f7fbbfa9-5ddc-4f6d-87e3-66f7234b09d5.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTcwMDc2NDgsIm5iZiI6MTc1NzAwNzM0OCwicGF0aCI6Ii8zMDkzMDY4OC80ODQyMjEyNzQtZjdmYmJmYTktNWRkYy00ZjZkLTg3ZTMtNjZmNzIzNGIwOWQ1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTA5MDQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwOTA0VDE3MzU0OFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTc0Y2NhMGE4ZDA2NWY4MDk1ODNkNjQ4YzA1NTZlNDYwNWI4YTExMWNlOGE0N2JlMTM1OTc3MDAwNmQ2MTk5NDAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.Rl05tJKWoi2YfLgC7WG5A04dTCfA1TKP8BUa9DgYgAY">
> 
> I mean seriously I'm not an English major lol. I use AI for writing basic stuff all the time, it writes better and modular than me

help or not AI sucks at stuff like this. it's just embarrasing. it's just AI slpo. it doesn't help anything. It would be fine if you simply asked it to fix spelling or grammar but this is just straight AI slop

acedmicabhishek commented:
> > > <img alt="image" width="857" height="893" src="https://private-user-images.githubusercontent.com/30930688/484221274-f7fbbfa9-5ddc-4f6d-87e3-66f7234b09d5.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTcwMDc2NDgsIm5iZiI6MTc1NzAwNzM0OCwicGF0aCI6Ii8zMDkzMDY4OC80ODQyMjEyNzQtZjdmYmJmYTktNWRkYy00ZjZkLTg3ZTMtNjZmNzIzNGIwOWQ1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTA5MDQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwOTA0VDE3MzU0OFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTc0Y2NhMGE4ZDA2NWY4MDk1ODNkNjQ4YzA1NTZlNDYwNWI4YTExMWNlOGE0N2JlMTM1OTc3MDAwNmQ2MTk5NDAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.Rl05tJKWoi2YfLgC7WG5A04dTCfA1TKP8BUa9DgYgAY">
> > 
> > I mean seriously I'm not an English major lol. I use AI for writing basic stuff all the time, it writes better and modular than me
> 
> help or not AI sucks at stuff like this. it's just embarrasing. it's just AI slpo. it doesn't help anything. It would be fine if you simply asked it to fix spelling or grammar but this is just straight AI slop

Sorry dude not trying to be rude or anything but seriously you guys are crying like a child , " ma'am he cheated from me" 
People already called me out for using ai , which I believe isn't wrong, literally everyone uses ai on prod.
I can also start calling out people for using high level language, saying "oh god this language sucks , using abstraction that slows down everything, cringe and also glorify C". 
Pretty idiotic right.
Stop being a Wimpy kid and move the fuck on.
And about that AI checker website 
It's a slop itself. Imagine saying "mkdir" is a ai written text, most of the readme content was written by me and enhanced via ai because I suck at writing.

supervoidcoder commented:
> > > > <img alt="image" width="857" height="893" src="https://private-user-images.githubusercontent.com/30930688/484221274-f7fbbfa9-5ddc-4f6d-87e3-66f7234b09d5.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTcwMDc2NDgsIm5iZiI6MTc1NzAwNzM0OCwicGF0aCI6Ii8zMDkzMDY4OC80ODQyMjEyNzQtZjdmYmJmYTktNWRkYy00ZjZkLTg3ZTMtNjZmNzIzNGIwOWQ1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTA5MDQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwOTA0VDE3MzU0OFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTc0Y2NhMGE4ZDA2NWY4MDk1ODNkNjQ4YzA1NTZlNDYwNWI4YTExMWNlOGE0N2JlMTM1OTc3MDAwNmQ2MTk5NDAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.Rl05tJKWoi2YfLgC7WG5A04dTCfA1TKP8BUa9DgYgAY">
> > > 
> > > 
> > > I mean seriously I'm not an English major lol. I use AI for writing basic stuff all the time, it writes better and modular than me
> > 
> > 
> > help or not AI sucks at stuff like this. it's just embarrasing. it's just AI slpo. it doesn't help anything. It would be fine if you simply asked it to fix spelling or grammar but this is just straight AI slop
> 
> Sorry dude not trying to be rude or anything but seriously you guys are crying like a child , " ma'am he cheated from me" People already called me out for using ai , which I believe isn't wrong, literally everyone uses ai on prod. I can also start calling out people for using high level language, saying "oh god this language sucks , using abstraction that slows down everything, cringe and also glorify C". Pretty idiotic right. Stop being a Wimpy kid and move the fuck on. And about that AI checker website It's a slop itself. Imagine saying "mkdir" is a ai written text, most of the readme content was written by me and enhanced via ai because I suck at writing.

Wow, no need to get all inappropriate and violent. You're the only one here that's acting like that. "Not to be rude" and start bombarding everything with "you guys are crying like a child" and "idiotic".
Irony aside, you absolutely don't need an english major to type something coherent. You can even make spelling mistakes! I was just saying that AI sucks because it can hallucinate. So what's the point of having a perfectly written README that's nonsensical at a second glance and makes mistakes? And the AI checker websites are wrong sometimes too, but you could just read the text yourself and figure it out yourself. If you just accept this word of advice, I just wanted to tell you that you should just write it yourself, even if it has mistakes or grammar. That's fine! People can make revisions later. But using AI makes it uncanny just to read, and is kind of a dead giveaway, and you can't really trust the text. If you HAVE to use AI, then... well, just personalize it to the best of your ability, like adding a joke, removing em dashes, or the flowery text with the double trait descriptors and the other AI shenanigany giveaways.

sammwyy commented:
Bruh, the point of using AI really isn't that bad, although for personal projects it feels a bit lame... and why the hell are you opening an issue to Felix? This is a personal repo, why are we even having this conversation here? Just behave.

The real criticism of AI wasn't just about using it, but that it seemed like a simple and poor attempt to be part of something—that is, to contribute to a personal repository. Hey, like I said, this is a personal repository. It's a bit of a shame for those trying to self-include. Don't fight over trivial things, just forget about it.

supervoidcoder commented:
> Bruh, the point of using AI really isn't that bad, although for personal projects it feels a bit lame... and why the hell are you opening an issue to Felix? This is a personal repo, why are we even having this conversation here? Just behave.
> 
> The real criticism of AI wasn't just about using it, but that it seemed like a simple and poor attempt to be part of something—that is, to contribute to a personal repository. Hey, like I said, this is a personal repository. It's a bit of a shame for those trying to self-include. Don't fight over trivial things, just forget about it.

I mean, pewdiepie probably intended for people to be able to contribute, otherwise he would have shut down issues tab and other stuff, or just made the entire repository private. Just because something is a personal project doesn't mean others can't or shouldn't contribute. But you're right in the sense that some people are contributing for the sake of contributing.
